### PR TITLE
Fix for eth/xdai balance symbol issue

### LINKED ISSUE #2369: [1.10] xDAI balance is displayed as ETH on the Claim screen
Reported in https://gnosisinc.slack.com/archives/C025G521XQD/p1643733022186869

xDAI balance is displayed as ETH on the Claim screen

**Expected**: balance in xDAI in Gnosis chain

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2369

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
I was trying to understand why it works on the header.
Well, that's because it's a cheat and it doesn't depend on the balance instance https://github.com/gnosis/cowswap/blob/30ea2ecb9a92c95622da487f825d36a6230612de/src/custom/components/Header/index.tsx#L222

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx [hunk 1]
```diff
@@ -411,7 +411,7 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
               <span>
                 <b>Balance:</b>
                 <i title={balance && `${formatMax(balance, balance.currency.decimals)} ${balance.currency.symbol}`}>
```
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:413]: why the title and the value displayed use different var? 
TITLE => balance.currency.decimals
VALUE => currencyAmount?.currency?.symbol

**Review by anxolin — APPROVED** [review]

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx (hunk 1 above)
Comment (alfetopito) [reply src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:413]: Ahhh good catch! Also need to update the title!

**Review by elena-zh — APPROVED** [review]
LGTM

**Pull request merged**